static/app/views/performance/transactionSummary/transactionSpans/spanDetails/content.tsx

```diff
@@ -234,7 +237,11 @@ function SpanDetailsHeader(props: HeaderProps) {
             ? formatPercentage(Math.min(frequency, totalCount) / totalCount)
             : '\u2014'}
         </SectionBody>
-        <SectionSubtext>{tn('%s event', '%s events', frequency)}</SectionSubtext>
+        <SectionSubtext>
+          {defined(avgOccurrences)
+            ? tct('[times] times per event', {times: avgOccurrences.toFixed(2)})
```

Comment:
```suggestion
            ? tct('[times] occurrences per event', {times: avgOccurrences.toFixed(2)})
```

Reply:
From slack, we decided that `occurrences` was too long for the copy here.